extensions/keycloak-authorization/runtime/src/main/java/io/quarkus/keycloak/pep/runtime/KeycloakPolicyEnforcerAuthorizer.java

```diff
@@ -29,6 +32,9 @@
 public class KeycloakPolicyEnforcerAuthorizer
         implements HttpSecurityPolicy, BiFunction<RoutingContext, SecurityIdentity, HttpSecurityPolicy.CheckResult> {
 
+    @Inject
```

Comment:
This injection is probably not needed yet as `quarkus-keycloak-authorization` will get the multi-tenancy support in a follow up PR ?